examples/dmd/dg_euler.py

```diff
@@ -0,0 +1,500 @@
+from pylibROM.algo import NonuniformDMD, AdaptiveDMD
+
+# 1. Initialize MPI.from mpi4py import MPI
+
+from mpi4py import MPI
+num_procs = MPI.COMM_WORLD.size
+myid = MPI.COMM_WORLD.rank
+
+parser = ArgParser(description='dg_euler')
+parser.add_argument('-m', '--mesh',
+                    default='periodic-square.mesh',
+                    action='store', type=str,
+                    help='Mesh file to use.')
+parser.add_argument('-p', '--problem',
+                    action='store', default=1, type=int,
+                    help='Problem setup to use. See options in velocity_function().')
+parser.add_argument('-rs', '--refine_serial',
+                    action='store', default=0, type=int,
+                    help="Number of times to refine the mesh uniformly before parallel.")
+parser.add_argument('-rp', '--refine_parallel',
+                    action='store', default=1, type=int,
+                    help="Number of times to refine the mesh uniformly after parallel.")
+parser.add_argument('-o', '--order',
+                    action='store', default=3, type=int,
+                    help="Finite element order (polynomial degree)")
+parser.add_argument('-s', '--ode_solver',
+                    action='store', default=4, type=int,
+                    help="ODE solver: 1 - Forward Euler,\n\t" +
+                    "            2 - RK2 SSP, 3 - RK3 SSP, 4 - RK4, 6 - RK6.")
+parser.add_argument('-tf', '--t_final',
+                    action='store', default=2.0, type=float,
+                    help="Final time; start time is 0.")
+parser.add_argument("-dt", "--time_step",
+                    action='store', default=-0.01, type=float,
+                    help="Time step.")
+parser.add_argument('-c', '--cfl_number',
+                    action='store', default=0.3, type=float,
+                    help="CFL number for timestep calculation.")
+parser.add_argument('-vis', '--visualization',
+                    action='store_true',
+                    help='Enable GLVis visualization')
+parser.add_argument('-visit', '--visit-datafiles',
+                    action='store_true', default=False,
+                    help='Save data files for VisIt (visit.llnl.gov) visualize.')
+parser.add_argument('-vs', '--visualization_steps',
+                    action='store', default=50, type=float,
+                    help="Visualize every n-th timestep.")
+
+# additional args for DMD
+parser.add_argument('-ef','--energy_fraction',
+                    action='store',default=0.9999, type=float,
+                    help='Energy fraction for DMD')
+parser.add_argument('-rdim','--rdim',
+                    action='store',default=-1,type=int,
+                    help='Reduced dimension for DMD')
+parser.add_argument('-crbf','--crbf',
+                    action='store',default=0.9,type=float,
+                    help='Closest RBF value')
+parser.add_argument('-nonunif','--nonunif','-no-nonunif','--no-nonunif',
```

Comment:
I tried to run it with this argument, but it fails with the following message:
```
dg_euler.py: error: argument -nonunif/--nonunif/-no-nonunif/--no-nonunif: expected one argument
```
I think `ArgParser` does not take arguments this way. You will probably have to split into `store_true` and `store_false` actions, with the same `dest`.